src/number.js

```diff
@@ -37,14 +38,16 @@ define([
  */
 Globalize.numberFormatter =
 Globalize.prototype.numberFormatter = function( options ) {
-	var cldr, maximumFractionDigits, maximumSignificantDigits, minimumFractionDigits,
-		minimumIntegerDigits, minimumSignificantDigits, pattern, properties;
+	var args, cldr, maximumFractionDigits, maximumSignificantDigits, minimumFractionDigits,
```

Comment:
This epic var list also looks dysfunctional. As a starting point to improve: All the min/maxXxx vars are used in one place, just once or maybe twice. All for validating `properties`. That would be trivial to extract into a separate function (in the same file).

Also without onvar most of these could be defined where they are actually used, making refactorings like the one described above much easier.